src/components/Popover/index.jsx

```diff
@@ -1,300 +1,66 @@
-    switch (interactionMode) {
-      case 'hover':
-        triggerProps.onMouseEnter = () => {
-          if (parseInt(delay, 10) > 0) {
-            hoverTimeout = setTimeout(setIsActive, delay, true);
-          } else {
-            setIsActive(true);
-          }
-        };
-        triggerProps.onMouseLeave = () => {
-          if (hoverTimeout) {
-            clearTimeout(hoverTimeout);
-          }
-          setIsActive(false);
-        };
-        triggerProps.onKeyUp = (e) => {
-          if (e.key === 'Tab') {
-            setIsActive(true);
-          }
-        };
-        triggerProps.onKeyDown = (e) => {
-          if (e.key === 'Tab') {
-            setIsActive(false);
-          }
-        };
-        triggerProps.tabIndex = '1';
-        break;
-      case 'click':
-        triggerProps.onClick = (e) => {
-          e.stopPropagation();
-          setIsActive(!isActive);
-        };
-        break;
-      default:
-        triggerProps = {};
-    }
-
-    const clonedTrigger = React.cloneElement(trigger, triggerProps);
-
-    // https://popper.js.org/popper-documentation.html#modifiers
-    const popperModifiers = [
-      {
-        name: 'offset',
-        enabled: true,
-        options: {
-          offset: [0, distance],
-        },
-      },
-      {
-        name: 'preventOverflow',
-        options: {
-          boundary: 'viewport',
-        },
-      },
-      {
-        name: 'computeStyles',
-        options: {
-          // When gpuAcceleration is enabled, `react-popper` always
-          // positions content to top/left 0 and uses `translate3d` to
-          // nudge it into place.
-          //
-          // This can cause issues with CSS transitions because it sets
-          // `will-change: transform`, which forces transitions to also
-          // ease the `translate3d`, which causes an unintended
-          // "diagonal slide" effect as the content appears.
-          gpuAcceleration: false,
-        },
-      },
-    ];
-
-    const contentStyle = {
-      zIndex: FDS.ZINDEX_MODAL,
-    };
-
-    const popperContent = (
-      <CSSTransition
-        onEntered={onOpen}
-        onExited={onClose}
-        in={isActive}
-        unmountOnExit
-        timeout={transitionName ? 200 : 0}
-        classNames={transitionName ? `rtg${transitionName}` : undefined}
+import hideOnEscFunc from './hideOnEsc';
+import triggerHelper from './triggerHelper';
+
+const Popover = ({
+  children,
+  isOpen,
+  placement = 'bottom-start',
+  trigger = 'click',
+  triggerElement,
+  onUserDismiss,
+  theme = 'default',
+  appendTo = () => document.body,
+}) => {
+  const hideOnEsc = useMemo(() => hideOnEscFunc(onUserDismiss), [onUserDismiss]);
+  const computedTrigger = triggerHelper(trigger, isOpen);
+  return (
+    <>
+      <Tippy
+        animation={false}
+        arrow={false}
+        visible={isOpen}
+        theme={`popover-${theme}`}
+        trigger={computedTrigger}
+        placement={placement}
```

Comment:
Looks like we may want to go ahead and implement Tippy's `offset` prop, with a public prop of `distance`. 

The `distance` prop is being used in a few places in `cbi-site`